### PR TITLE
Nebula nicknames: Add name for some bright objects

### PR DESCRIPTION
**Pull request by silas1037**
Add name for NGC6397/C86 in Ara (astrobin)
One of the nearest globular clusters but nameless; Vt5.2, bright enough for naming 'Constellation Globular Clusters'

### CODE REVIEW

**Comment by silas1037** [comment]
Please wait and I would like to add some names

**Comment by silas1037** [comment]
add name for NGC1313 from WP

**Comment by silas1037** [comment]
'Bubble Galaxy' for NGC3521 is from searching, @alex-w  can you help me to add a fitable source for it?

**Comment by silas1037** [comment]
I guest it is a typo to ACO  S737. Fornax Cluster and Fornax Group should be the same thing.

>ACO  S373            _("Fornax Cluster") # NED, WP, BCH, WSO, WSG
ACO  S737            _("Fornax Group") # SIMBAD
ACO  S737            _("NGC 1399 Group") # WP, SIMBAD

**Comment by alex-w** [comment]
> 'Bubble Galaxy' for NGC3521 is from searching, @alex-w can you help me to add a fitable source for it?

Probably [ESA/Hubble](https://esahubble.org/images/potw1538a/) and [AAPOD2](https://www.aapod2.com/blog/ngc-3521)

**Comment by silas1037** [comment]
> > 'Bubble Galaxy' for NGC3521 is from searching, @alex-w can you help me to add a fitable source for it?
> 
> Probably [ESA/Hubble](https://esahubble.org/images/potw1538a/) and [AAPOD2](https://www.aapod2.com/blog/ngc-3521)

Ok, thanks.

**Comment by silas1037** [comment]
Already change to ACO S373 according to simbad

**Comment by silas1037** [comment]
@alex-w  How about adding name 'Aphrodite's jewel' for NGC1398 ? (https://www.astrobin.com/1vm4dq/B/)

File: nebulae/default/names.dat [hunk 1]
```diff
@@ -402,7 +403,7 @@ NGC  3372            _("Homunculus Nebula") # WK, WSO
 NGC  3372            _("Keyhole Nebula") # WK, SIMBAD, BCH, SG, WSO, DN, DAN
 NGC  3372            _("The Great Nebula in Carina") # DN, APOD
 NGC  3432            _("Knitting Needle Galaxy") # B500, SD
-NGC  3521            _("Bubble Galaxy") # 
+NGC  3521            _("Bubble Galaxy") # AAPOD2
```
Comment (alex-w) [inline nebulae/default/names.dat:406]: Please add ESA/Hubble source as primary also

**Comment by alex-w** [comment]
> @alex-w How about adding name 'Aphrodite's jewel' for NGC1398 ? (https://www.astrobin.com/1vm4dq/B/)

I disagree, because this is single source. @gzotti ?

File: nebulae/default/names.dat (hunk 1 above)
Comment (silas1037) [reply nebulae/default/names.dat:406]: no abbrevation code for ESA/Hubble source?

Comment (alex-w) [reply nebulae/default/names.dat:406]: Well, ESA/Hubble source is similar to APOD and AAPOD2, so, you need introduce new abbreviation for it (probably scheme will be similar to APOD)

Comment (silas1037) [reply nebulae/default/names.dat:406]: ok

**Comment by silas1037** [comment]
> 

Ok, just ignore it.

**Comment by silas1037** [comment]
alright, I have finished this pr.

**Review by alex-w — APPROVED** [review]

**Comment by gzotti** [comment]
> > @alex-w How about adding name 'Aphrodite's jewel' for NGC1398 ? (https://www.astrobin.com/1vm4dq/B/)
> 
> I disagree, because this is single source. @gzotti ?

I would not use any singular names from personal websites or even new name creations from "Hubble picture of the day"-type sites when they do not cause enough response within 1 year to be generally accepted and used. Who will ever use or remember all those names? Many people also despise the APOD naming fantasies.

**Pull request merged**